base_layer/key_manager/src/key_manager.rs

```diff
@@ -77,7 +77,12 @@ where
 
     /// Derive a new private key from master key: derived_key=SHA256(master_key||branch_seed||index)
     pub fn derive_key(&self, key_index: u64) -> Result<DerivedKey<K>, ByteArrayError> {
-        let concatenated = format!("{}{}", self.seed.entropy().to_vec().to_hex(), key_index);
```

Comment:
@philipr-za does this indicate the current wallets are all deriving on the same branch??

Reply:
Ja it would! This is a good catch.